packages/webview_flutter/android/src/main/java/io/flutter/plugins/webviewflutter/FlutterWebViewClient.java

```diff
@@ -32,8 +34,7 @@ void setHasNavigationDelegate(boolean hasNavigationDelegate) {
   }
```

Comment:
This can be called after the initial call to createWebViewClient (e.g the app starts without a navigation delegate and later rebuilds the webview with a navigation delegate)

Reply:
I was wondering about this. I thought the `WebView` would be a constant widget, but realized that platform view do rebuild on state changes. I'll update the PR.